peerswap/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: peerswap_web_1
+      APP_PORT: 1984
+
+  web:
+    image: ghcr.io/impa10r/peerswap-web:v1.3.5@sha256:6897ca22aebbd121c55513be9fa0cabe5b05349e12b28b583f3dcc46e5ed7545
+    user: "1000:1000"
+    restart: on-failure
+    stop_grace_period: 1m
+    environment:
+      # App
+      NETWORK: $APP_BITCOIN_NETWORK
+      # LND
+      LND_HOST: ${APP_LIGHTNING_NODE_IP}:${APP_LIGHTNING_NODE_GRPC_PORT}
+      LND_MACAROONPATH: /home/peerswap/.lnd/data/chain/bitcoin/${APP_BITCOIN_NETWORK}/admin.macaroon
+      # Elements RPC
+      ELEMENTS_USER: elements
+      ELEMENTS_PORT: $APP_ELEMENTS_NODE_RPC_PORT
+      ELEMENTS_HOST: http://elements_node_1
+      ELEMENTS_WALLET: peerswap
+      ELEMENTS_FOLDER: /home/elements/.elements
+      ELEMENTS_FOLDER_MAPPED: /home/peerswap/.elements
+    volumes:
+      - ${APP_DATA_DIR}/data:/home/peerswap/.peerswap
+      - ${APP_LIGHTNING_NODE_DATA_DIR}:/home/peerswap/.lnd:ro
+      - /home/umbrel/umbrel/app-data/elements/data:/home/peerswap/.elements:ro
```

Comment:
To make sure this works for all users, including those who are running pre-1.0 umbrelOS on custom linux installs, we should export the elements core data directory as an environment variable from the PeerSwaps exports.sh, and define it as a relative path from the script.

You would do something like this (untested):
```bash
ELEMENTS_DATA_DIR="$(readlink -f $(dirname "${BASH_SOURCE[0]}")/..)/elements/data"
```

Right now, a custom linux user on umbrelOS 0.5.x might have their umbrel install at a path other than `/home/umbrel/umbrel`. For example, it's possible that their elements data directory would be at something like `/home/skye/installs/umbrel/app-data/elements/data`. If PeerSwap hardcodes `/home/umbrel/umbrel/app-data/elements/data` then this user wouldn't be able to use the app.